wire-ios-data-model/Source/Model/Message/ZMClientMessage+Encryption.swift

```diff
@@ -23,25 +23,6 @@ private var zmLog = ZMSLog(tag: "message encryption")
 
 public let ZMFailedToCreateEncryptedMessagePayloadString = "💣"
 
-// MARK: - Encrypted data for recipients
```

Comment:
this is all the old code path to encrypt message which is not needed, this is removed only now, because tests needed to be updated heavily